plugins/onepad/Linux/ini.cpp

```diff
@@ -106,12 +109,12 @@ void LoadConfig()
     if (fscanf(f, "ff_intensity = %u\n", &value) == 1)
         g_conf.set_ff_intensity(value);
 
-    size_t uid;
-    if (fscanf(f, "uid[0] = %zu\n", &uid) == 1)
-        g_conf.set_joy_uid(0, uid);
-    if (fscanf(f, "uid[1] = %zu\n", &uid) == 1)
-        g_conf.set_joy_uid(1, uid);
-
+	size_t uid;
+	for (int uidpadload = 0; uidpadload < GAMEPAD_NUMBER; uidpadload++)
+		if (fscanf(f, "uid(%d) = %zu\n", uidpadload, &uid) == 1) {
+			g_conf.set_joy_uid(uidpadload, uid);
+		}
+	
```

Comment:
Whitespace.

Reply:
Lol, nice catch, fixed, will send with next set of tests.